### DISCUSSION TITLE
How do I provision a single 'service' in the azure.yaml file?

### DISCUSSION THREAD

**Discussion by diberry** [Azure/azure-dev#2649] [Q&A]
For example, suppose I'm a new dev to the [contoso real estate](https://github.com/azure-Samples/contoso-real-estate/) app. How do I provision only the [cms](https://github.com/Azure-Samples/contoso-real-estate/blob/main/azure.yaml#L35C1-L35C1) service?

**Comment by savannahostrowski** [comment]
We have no gesture to provision a specific service at this time - I've converted this to a feature request https://github.com/Azure/azure-dev/issues/2650
